dom.bs

```diff
@@ -1201,6 +1201,9 @@ for discussion).
  <a>relatedTarget</a> against <var>target</var> if <var>event</var>'s <a>relatedTarget</a> is
  non-null, and null otherwise.
 
+ <li><p>If <var>event</var>'s {{Event/isTrusted}} attribute is true and <var>target</var> is
+ <var>relatedTarget</var>, then terminate these steps.
```

Comment:
No, I don't think we should be specifying this in the DOM specification. Also, this would affect every UI event dispatch beyond just ones inside shadow trees, and I'm not okay with that.

Please don't merge this.